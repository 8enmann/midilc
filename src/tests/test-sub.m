/* Test subtraction
   Tests subtracting a duration from a note */

main() {
  Note a;
  Sequence b;
  Chord c;
  b = new_sequence();
  a = A3q;
  a.duration = a.duration - 1;
  c = new_chord(a);
  b = b + c;
  b = b + b;
  play(b);
}