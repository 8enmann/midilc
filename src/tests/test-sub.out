0,3,57
3,3,57

